UserAgent/Views/Privacy Indicator/PrivacyDashboardUtils.swift

```diff
@@ -0,0 +1,139 @@
+import Shared
+
+enum PrivacyDashboardUtils {
+    static func stackViewForStatEmpty(
+        _ color: UIColor,
+        _ text: String
+    ) -> UIStackView {
+        let dot = dotView(color)
+        let label = statLabel(text)
+
+        let stackView = UIStackView(arrangedSubviews: [dot, label])
+        stackView.spacing = 5
+        stackView.alignment = .top
+        return stackView
+    }
+
+    static func stackViewForStat(
+        _ color: UIColor,
+        _ text: String,
+        _ value: String
+    ) -> UIStackView {
+        let dot = dotView(color)
```

Comment:
I see this pattern of private methods creating views very often. Wonder how it would look like if instead a more object-oriented approach would be used - let me explain. Instead of using a generator function we could have a private class subclassing UIView that would create a view of given properties. This would probably make the UIKit code to look much more like SwiftUI. What do you think?

Reply:
make sense; it will be not like a swiftui :), but good catch (I ll try)